Kudu.Services/ServiceHookHandlers/DropboxHandler.cs

```diff
@@ -37,7 +38,7 @@ public DeployAction TryParseDeploymentInfo(HttpRequestBase request, JObject payl
                 dropboxInfo = DropboxInfo.CreateV1Info(payload, GetRepositoryType());
                 message = String.Format(CultureInfo.CurrentUICulture, Resources.Dropbox_SynchronizingNChanges, dropboxInfo.DeployInfo.Deltas.Count);
             }
-            else if (String.Equals(payload.Value<string>("scmType"), "DropboxV2", StringComparison.OrdinalIgnoreCase))
+            else if (String.Equals(payload.Value<string>(DropboxVersionKey), "2", StringComparison.OrdinalIgnoreCase))
```

Comment:
Since we will not (don't need to) introduce a new ScmType, I explicitly using dropbox_version in the payload to signify the different version.

Reply:
Is no one using an scmType of `DropboxV2`?

Reply:
Correct.   BTW, this was done long time ago (but not completely correct) to anticipate the Dropbox V2.   Changing ScmType is unnecessary and may involve more codes than we need.